modules/ROOT/pages/cache-scope.adoc

```diff
@@ -385,7 +385,31 @@ There are two message processors for invalidating caches:
 <ee:invalidate-key cachingStrategy-ref="InvalidatableCachingStrategy" keyGenerator-ref="MD5MuleEventKeyGenerator"/>
 ----
 
-== Example
+[IMPORTANT]
+====
+Studio does not have the Invalidate Cache component. You need to manually add the component by modifying the configuration XML, adding an `ee:invalidate-cache` element to your flow.
+====
+
+=== Example: Invalidating Cache Usage
+
+To invalidate a cache, you can create a Flow with two components:
+
+* An `http:listener` component that will be the entry point of the Flow.
```

Comment:
`An` -> `A`

Reply:
An is correct because of the way the h is pronounced.